src/util.ts

```diff
@@ -269,6 +270,78 @@ class Util {
     }
 
     /**
+     * Removes the trailing `Brightscript Debugger>` prompt if present. If not present, returns original value
+     * @param value
+     */
+    public trimDebugPrompt(value: string) {
+        const match = /(.*?)\r?\nBrightscript Debugger>\s*/is.exec(value);
+        if (match) {
+            return match[1];
+        } else {
+            return value;
+        }
+    }
+
+    /**
+     * Get the keys for a given variable expression, or undefined if the expression doesn't make sense.
+     */
+    public getVariablePathOld(expression: string): string[] {
```

Comment:
Let's delete this function since we don't need it. We can always find it again in git history if necessary.

Reply:
done